contracts/CapabilityProxy.cdc

```diff
@@ -15,10 +15,11 @@ pub contract CapabilityProxy {
     pub let PrivatePath: PrivatePath
     pub let PublicPath: PublicPath
     
+    /* Events */
+    //
     pub event ProxyCreated(id: UInt64)
-    
-    pub event CapabilityAdded(address: Address, type: Type, isPublic: Bool)
-    pub event CapabilityRemoved(address: Address, type: Type)
+    pub event CapabilityAdded(id: UInt64, type: Type, isPublic: Bool) // TODO: Decide on Type or identifier String
+    pub event CapabilityRemoved(id: UInt64, type: Type) // TODO: Decide on Type or identifier String
```

Comment:
I second this, but it is not very important for me. Flowscan show types very verbose m. Just check a storefront listing.

Reply:
Identifier is more friendly, but it is also more prescriptive in that it prevents anyone who would want to listen to the type from doing so. On the other hand, emitting the event lets anyone who wants to grab the identifier get to it and just replace the field. Because of that, I'd stick with type just so that we don't prevent folks from doing what they want, here. Thoughts?

Reply:
Agreed. Better to have a flexible solution